createrepo_c.spec

```diff
@@ -10,13 +10,13 @@
 %bcond_without drpm
 %endif
 
-%if 0%{?rhel} || 0%{?fedora} < 29
+%if 0%{?rhel} || (0%{?fedora} && 0%{?fedora} < 29)
```

Comment:
Does it even make sense to care about so old Fedora? I mean
```
%if 0%{?rhel}
```
might be enough.

Reply:
Good point, removed.